### ISSUE TITLE
Question About Hyperparameter Search

### ISSUE THREAD

**Issue by marinadunn** [marinadunn/europa-chaos-ML#4]
Hi Ahmed, 

I am working on consolidating the code and writing documentation for reproducibility. I had a question about the hyperparameter search: for the optimal parameters we listed (learning rate of 0.001, batch size of 1, window crop size of 250x250, stride of 64, Adam optimizer, training with 3 trainable backbone layers, and training duration of 15 epochs), to confirm- was this with the average F1 score objective function (`avg_f1_objective`) or something else? And what was the `min_sheet_area` value used for that? Thanks!
